### PR TITLE
added /jerome

### PR DESCRIPTION
**Pull request by jeromepalayoor**

### CODE REVIEW

File: main.py
Comment (Mini-Ware) [inline main.py:327]: This line doesn't seem very nice, I think something needs to change here before merging

**Review by Mini-Ware — CHANGES_REQUESTED** [review]
line 327 should be replaced with "/mini is still better", apart from that everything else looks good

File: main.py
Comment (jeromepalayoor) [reply main.py:327]: ok

Comment (jeromepalayoor) [reply main.py]: updated!

Comment (Mini-Ware) [reply main.py]: alright, now we wait for jus to review

File: main.py
Comment (Jus-Codin) [inline main.py:309]: We should not use the requests library as it is blocking, if anyone runs this command, all other users have to wait until this command finishes running before their commands can be run. This would also affect mushroom farming as well.
Refer to [this](https://discordpy.readthedocs.io/en/stable/faq.html#what-does-blocking-mean) for more info and how to solve it

**Review by Jus-Codin — CHANGES_REQUESTED** [review]

**Comment by jeromepalayoor** [comment]
i have updated it to use asyncio

**Review by Jus-Codin — APPROVED** [review]
looks good to me, pulling it

**Pull request merged**
